### DISCUSSION TITLE
How can I run single sets from the CLI?

### DISCUSSION THREAD

**Discussion by ADoebeling** [rectorphp/rector#8684] [Q&A] [answered]
I'd like to run the sets in sequence from CLI. Is there any way to do this without re-generating the rector.php config file repeatedly?

In Psalm, this is possible by executing:

```bash
php vendor/bin/psalm --alter --issues=MissingClosureReturnType
php vendor/bin/psalm --alter --issues=InvalidReturnType
php vendor/bin/psalm --alter --issues=InvalidNullableReturnType
...
```

Thanks in advance!
Andreas.

**Comment by mspirkov** [comment, marked as answer]
I think this might help you: https://getrector.com/documentation/run-single-rule

You can also make multiple configurations and use them like this:

```bash
./vendor/bin/rector process --dry-run --config=rector.php
```
